secrets-yaml-reference_5b680e022c7d3a03f89d6261.md

```diff
@@ -0,0 +1,103 @@
+* [About Secrets YAML reference](#secrets-yaml-reference)
+* [Properties](#properties)
+   * [apiVersion](#apiversion)
+   * [kind](#kind)
+   * [metadata](#metadata)
+      * [name](#name-in-metadata)
+   * [data](#data)
+   * [env_vars](#env_vars)
+      * [name](#name-in-env_vars)
+      * [value](#value)
+* [Secrets example](#secrets-example)
+* [See also](#see-also)
+
+# Secrets YAML Reference
+
+This document is the reference for the YAML grammar used for creating `secrets`
+buckets for Semaphore 2.0 projects.
```

Comment:
This document is the reference for the YAML grammar used for creating secrets.

Reply:
Done.